translate/storage/test_po.py

```diff
@@ -749,7 +722,8 @@ def test_id(self):
         # commented out for conformance to gettext.
 #        assert pofile.units[4].getid() == "tree\0trees"
 
-    def test_non_ascii_header_comments(self):
+    # method with same name below?
+    def __test_non_ascii_header_comments(self):
```

Comment:
i think with this function we should either:
- remove it altogether, or
- integrate what is there into the test below